code/controllers/subsystem/minor_mapping.dm

```diff
@@ -1,4 +1,5 @@
 #define PROB_MOUSE_SPAWN 98
+#define PROB_SPIDER_REPLACEMENT 50
```

Comment:
you forgot to undef this. i've already called in some coconut crabs to crush your skull